packages/react/README.md

```diff
@@ -264,6 +318,108 @@ const [
 ] = useSetChain()
 ```
 
+## `useNotifications`
+
+This hook allows the dev to access all notifications if enabled, send custom notifications and update notify >enable/disable and change transactionHandler function<
```

Comment:
@Adamj1232 how did you intend for this to render:
`>enable/disable and change transactionHandler function<`

It looks like this in markdown:

This hook allows the dev to access all notifications if enabled, send custom notifications and update notify >enable/disable and change transactionHandler function<